egui/src/containers/scroll_area.rs

```diff
@@ -493,14 +493,17 @@ impl Prepared {
                     let clip_rect = content_ui.clip_rect();
                     let visible_range = min..=min + clip_rect.size()[d];
 
+                    // if the ui is too big to completely fit in the scroll view, align it to the left/top
```

Comment:
Is this the best behavior?

Another alternative is to scroll the minimum amount that makes the scroll view fill with the ui-range. So for instance, if we want to scroll to an image, and that image is very big then we either:

* scroll so that the top of the image is at the top of the scroll view
* scroll so that the bottom of the image is at the bottom of the scroll view (if that is closer)
* don't scroll at all, if all we are currently seeing is the image

Reply:
Yeah, I like the simmetry. I fixed it to have that behavior :point_up: